### ISSUE TITLE
sensors output does not show any fan information or asic temperature on s6100

### ISSUE THREAD

**Issue by lguohan** [Azure/sonic-platform-modules-dell#6]
```
root@str-s6100-acs-1:/home/admin# sensors
coretemp-isa-0000
Adapter: ISA adapter
Core 0:       +29.0 C  (high = +98.0 C, crit = +98.0 C)
Core 1:       +29.0 C  (high = +98.0 C, crit = +98.0 C)
Core 2:       +31.0 C  (high = +98.0 C, crit = +98.0 C)
Core 3:       +31.0 C  (high = +98.0 C, crit = +98.0 C)
```

**Comment by lguohan** [comment]
here is the sensor dump on s6000.

"""
admin@str-s6000-acs-7:~$ sensors
acpitz-virtual-0
Adapter: Virtual device
temp1:        +26.8 C  (crit = +127.0 C)
temp2:        +26.8 C  (crit = +175.0 C)

coretemp-isa-0000
Adapter: ISA adapter
Core 0:       +27.0 C  (crit = +95.0 C)
Core 1:       +28.0 C  (crit = +95.0 C)

w83627dhg-isa-0a00
Adapter: ISA adapter
VCore 1:      +1.17 V  (min =  +0.00 V, max =  +1.74 V)
VCore 2:      +0.86 V  (min =  +0.00 V, max =  +1.74 V)
AVCC:         +3.36 V  (min =  +2.98 V, max =  +3.63 V)
+3.3V:        +3.36 V  (min =  +2.98 V, max =  +3.63 V)
3VSB:         +3.38 V  (min =  +2.98 V, max =  +3.63 V)
Vbat:         +3.28 V  (min =  +2.70 V, max =  +3.63 V)

jc42-i2c-10-18
Adapter: i2c-2-mux (chan_id 0)
temp1:        +31.2 C  (low  =  +0.0 C)
                       (high = +50.0 C, hyst = +50.0 C)
                       (crit = +85.0 C, hyst = +85.0 C)

emc1403-i2c-10-4d
Adapter: i2c-2-mux (chan_id 0)
temp1:        +27.0 C  (low  =  +0.0 C)
                       (high = +50.0 C, hyst = +40.0 C)
                       (crit = +85.0 C, hyst = +75.0 C)
temp2:        +30.0 C  (low  =  +0.0 C)
                       (high = +50.0 C, hyst = +40.0 C)
                       (crit = +85.0 C, hyst = +75.0 C)
temp3:        +31.0 C  (low  =  +0.0 C)
                       (high = +50.0 C, hyst = +40.0 C)
                       (crit = +85.0 C, hyst = +75.0 C)

max6620-i2c-11-29
Adapter: i2c-2-mux (chan_id 1)
fan1:        15360 RPM  (div = 2)
fan2:        15360 RPM  (div = 2)
fan3:        15360 RPM  (div = 2)
fan4:        15360 RPM  (div = 2)

max6620-i2c-11-2a
Adapter: i2c-2-mux (chan_id 1)
fan1:        15855 RPM  (div = 2)
fan2:        14894 RPM  (div = 2)

ltc4215-i2c-11-40
Adapter: i2c-2-mux (chan_id 1)
in1:         +11.93 V  
in2:         +11.98 V  
power1:        7.21 W  
curr1:        +0.60 A  

ltc4215-i2c-11-42
Adapter: i2c-2-mux (chan_id 1)
in1:         +11.93 V  
in2:         +12.10 V  
power1:        6.30 W  
curr1:        +0.53 A  

tmp75-i2c-11-4c
Adapter: i2c-2-mux (chan_id 1)
temp1:        +31.6 C  (high = +50.0 C, hyst = +25.0 C)

tmp75-i2c-11-4d
Adapter: i2c-2-mux (chan_id 1)
temp1:        +25.4 C  (high = +50.0 C, hyst = +25.0 C)

tmp75-i2c-11-4e
Adapter: i2c-2-mux (chan_id 1)
temp1:        +26.1 C  (high = +50.0 C, hyst = +25.0 C)

dni_dps460-i2c-1-58
Adapter: SMBus iSMT adapter at ff781000
fan1:        14688 RPM
temp1:        +23.5 C  (high = +50.0 C)
temp2:        +33.5 C  (high = +50.0 C)

dni_dps460-i2c-1-59
Adapter: SMBus iSMT adapter at ff781000
fan1:        14720 RPM
temp1:        +23.5 C  (high = +50.0 C)
temp2:        +31.5 C  (high = +50.0 C)
"""

**Comment by lguohan** [comment]
resolved in #15
